src/UserSettingsPage.cpp

```diff
@@ -2127,11 +2127,25 @@ UserSettingsModel::exportSessionKeys()
     if (!ok)
         return;
 
+    auto repeatedPassword = QInputDialog::getText(nullptr,
+                                                  tr("Repeat File Password"),
+                                                  tr("Repeat the passphrase:"),
+                                                  QLineEdit::Password,
+                                                  QLatin1String(""),
+                                                  &ok);
+    if (!ok)
+        return;
+
     if (password.isEmpty()) {
         QMessageBox::warning(nullptr, tr("Error"), tr("The password cannot be empty"));
         return;
     }
 
+    if (password != repeatedPassword) {
+        QMessageBox::warning(nullptr, tr("Error"), tr("Passwords don't match"));
+        return;
+    }
+
```

Comment:
We may want to warn that the repeated password is empty if they've entered a password.
This should be obvious to the end user anyways, but
```suggestion
    if (password.isEmpty()) {
        QMessageBox::warning(nullptr, tr("Error"), tr("The password cannot be empty"));
        return;
    }

    if (repeatedPassword.isEmpty()) {
        QMessageBox::warning(nullptr, tr("Error"), tr("Please repeat the password"));
        return;
    }

    if (password != repeatedPassword) {
        QMessageBox::warning(nullptr, tr("Error"), tr("Passwords don't match"));
        return;
    }

```